mm2src/mm2_p2p/src/behaviours/atomicdex.rs

```diff
@@ -714,23 +713,12 @@ fn start_gossipsub(
             .map_err(|e| AdexBehaviourError::InitializationError(e.to_owned()))?;
 
         // build a gossipsub network behaviour
-        let mut gossipsub = Gossipsub::new(MessageAuthenticity::Author(local_peer_id), gossipsub_config)
+        let gossipsub = Gossipsub::new(MessageAuthenticity::Author(local_peer_id), gossipsub_config)
             .map_err(|e| AdexBehaviourError::InitializationError(e.to_owned()))?;
 
         let floodsub = Floodsub::new(local_peer_id, config.netid != DEFAULT_NETID);
 
-        let mut peers_exchange = PeersExchange::new(network_info);
-        if !network_info.in_memory() {
-            // Please note WASM nodes don't support `PeersExchange` currently,
-            // so `get_all_network_seednodes` returns an empty list.
-            for (peer_id, addr, _domain) in get_all_network_seednodes(config.netid) {
-                let multiaddr = addr.try_to_multiaddr(network_info)?;
-                peers_exchange.add_peer_addresses_to_known_peers(&peer_id, iter::once(multiaddr).collect());
-                if peer_id != local_peer_id {
-                    gossipsub.add_explicit_relay(peer_id);
```

Comment:
Why `add_explicit_relay` was removed? Maybe we can have it as part of config instead for each seed in `seednodes` list, what do you think?

Reply:
They were used for hard-coded seeds only and we don't have them anymore. How do you want to handle that at config level? If you change `seednodes` type it will cause BC.

Reply:
> They were used for hard-coded seeds only and we don't have them anymore.

What GUI will pass to the `seednodes` config will be the same as the removed hardcoded seeds, I just want to be sure if we will get the same behaviour as before or it will be better/worse.

Reply:
>  How do you want to handle that at config level? If you change seednodes type it will cause BC.

It can be a new config parameter that is applied to the whole list.

Reply:
It's not going to worse if you configure seednodes properly. Explicit relay was some kind of `keep these connected all the time in case if we have trouble with others`.

In my opinion this is a cheap hack/workaround.

Reply:
I understand and agree with this being a cheap hack/workaround. But with the dex network down and not enough seeds to test this change with, can we know for sure if this will work fine.

Reply:
I pinged QA team to ask for the coverage for this.

Reply:
dex network is actually fine, else all this swaps would not be there: https://www.coingecko.com/en/exchanges/komodo-wallet
there are though only 2 seeds with WSS that i know of (seed01.kmdefi.net, seed02.kmdefi.net), all the others have only TCP

Reply:
Resolving this as seednodes passed through config are now added as explicit relays. This will lead to the same behaviour as before when passing the previously hardcoded nodes through config.

Reply:
> Resolving this as seednodes passed through config are now added as explicit relays. This will lead to the same behaviour as before when passing the previously hardcoded nodes through config.

It's also that this behavior was missing in the previous approach. If someone configures the `seednodes` parameter, all of them should be treated as explicit relays since they are **explicitly** defined in the configuration.